yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/identity/IdentityService.java

```diff
@@ -0,0 +1,98 @@
+package com.yoti.api.client.spi.remote.call.identity;
+
+import static com.yoti.api.client.spi.remote.call.HttpMethod.HTTP_POST;
+import static com.yoti.api.client.spi.remote.call.YotiConstants.AUTH_ID_HEADER;
+import static com.yoti.api.client.spi.remote.call.YotiConstants.CONTENT_TYPE;
+import static com.yoti.api.client.spi.remote.call.YotiConstants.CONTENT_TYPE_JSON;
+import static com.yoti.api.client.spi.remote.call.YotiConstants.DEFAULT_IDENTITY_URL;
+import static com.yoti.api.client.spi.remote.call.YotiConstants.PROPERTY_YOTI_API_URL;
+import static com.yoti.api.client.spi.remote.util.Validation.notNull;
+
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.net.URISyntaxException;
+import java.nio.charset.StandardCharsets;
+import java.security.GeneralSecurityException;
+import java.security.KeyPair;
+
+import com.yoti.api.client.identity.SessionCreation;
+import com.yoti.api.client.identity.SessionResult;
+import com.yoti.api.client.spi.remote.call.ResourceException;
+import com.yoti.api.client.spi.remote.call.SignedRequest;
+import com.yoti.api.client.spi.remote.call.SignedRequestBuilderFactory;
+import com.yoti.api.client.spi.remote.call.factory.UnsignedPathFactory;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class IdentityService {
+
+    public static IdentityService newInstance() {
+        return new IdentityService(
+                new ObjectMapper(),
```

Comment:
The `ObjectMapper` should be configured so that it only considers explicitly declared `JsonProperty` annotated fields, to avoid "accidental" (de)serialisation due to a getter or setter for example.

It should also probably be configured to ignore `null` fields (go look at the config we use in connect's `JsonConfig`)